src/electron/views/report/unified-report-name-generator.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { ReportNameGenerator } from 'reports/report-name-generator';
+
+export class UnifiedReportNameGenerator extends ReportNameGenerator {
```

Comment:
We should favor composition over inheritance and I feel like this case lends itself well to composition. Let's discuss offline.